mixer/cmd/mixs/cmd/server.go

```diff
@@ -77,6 +77,8 @@ func serverCmd(info map[string]template.Info, adapters []adapter.InfoFn, printf,
 
 	serverCmd.PersistentFlags().BoolVar(&sa.UseAdapterCRDs, "useAdapterCRDs", sa.UseAdapterCRDs,
```

Comment:
i wonder: will we ever want to support a mix of these cases moving forward?

Reply:
Probably not for actual users. We can delete them both together in the next release. The internal tests and test configs use a mix.